src/Core/AdminConsole/Services/Implementations/OrganizationService.cs

```diff
@@ -591,9 +591,9 @@ await _referenceEventService.RaiseEventAsync(
             else
             {
                 await _paymentService.PurchaseOrganizationAsync(organization, signup.PaymentMethodType.Value,
```

Comment:
❌  My understanding of this feature is that we want to be able to start an organization subscription without a payment method. The underlying code for the `OrganizationBillingService.Finalize(Sale)` handles that, but when the Stripe deprecation feature flag is off, I don't think the `StripePaymentService` will handle that. The [method signature](https://github.com/bitwarden/server/blob/3c86ec6a35fd6b4995214124e0d5b8eb356ffd5c/src/Core/Services/Implementations/StripePaymentService.cs#L54-L57) actually demands a non-nullable `PaymentMethodType`, which is why it's being passed `signup.PaymentMethodType.Value`. But if your request is no longer validating that the incoming `PaymentMethodType` is actually provided, by design, then you're going to get a null reference exception here.